pkgs/development/python-modules/aioslimproto/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "aioslimproto";
+  version = "1.0.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "home-assistant-libs";
+    repo = pname;
+    rev = version;
+    hash = "sha256-kR2PG2eivBfqu67hXr8/RRvo5EzI75e8NmG15NPGo1E=";
+  };
+
+  # Module has no tests
+  doCheck = false;
```

Comment:
Well, no but they are empty...

https://github.com/home-assistant-libs/aioslimproto/tree/main/tests

Reply:
I guess that the tests are coming.